build/jest/react-intl-mock.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export const intlMock = {
+    formatMessage: message => message.defaultMessage || message.message,
+    formatDate: date => date,
+};
+
+export const FormattedDate = () => <div />;
+FormattedDate.displayName = 'FormattedDate';
+
+export const FormattedTime = () => <div />;
+FormattedTime.displayName = 'FormattedTime';
+
+export const FormattedMessage = () => <div />;
```

Comment:
Do these need to take any props?

Reply:
I pulled this from box-annotations where it seems to be working okay as-is.